src/main/java/com/epam/ta/reportportal/core/bts/handler/impl/UpdateBugTrackingSystemHandlerImpl.java

```diff
@@ -145,22 +134,55 @@ public OperationCompletionRS updateBugTrackingSystem(UpdateBugTrackingSystemRQ u
 	}
 
 	@Override
-	public OperationCompletionRS integrationConnect(UpdateBugTrackingSystemRQ updateRequest, Long integrationId,
+	public OperationCompletionRS integrationConnect(BtsConnectionTestRQ connectionTestRQ, Long integrationId,
```

Comment:
maybe we should name the method according what it is doing
for example, testIntegrationConnection

Reply:
done